tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -7769,7 +7769,7 @@ SpirvInstruction *SpirvEmitter::processWaveBroadcast(const CallExpr *callExpr) {
   const QualType retType = callExpr->getCallReturnType(astContext);
   if (numArgs == 2)
```

Comment:
Nit: why don't we add a comment "WaveReadLaneAt should be translated to OpGroupNonUniformShuffle, not OpGroupNonUniformBroadcast." ?

Reply:
SG. Done.